### PR TITLE
refactor: sidebar transition to open and close in mobile

### PR DESCRIPTION
**Pull request by omateusamaral**
In this pull request I added a transition to open and close the sidebar in mobile version.

## Video

[Gravação de tela de 15-04-2024 19:54:00.webm](https://github.com/ocodista/trampar-de-casa/assets/37390930/0b04a195-5a1c-4128-8564-04b14717eebb)

### CODE REVIEW

File: apps/web/app/components/Header.tsx [hunk 1]
```diff
@@ -72,62 +72,64 @@ export const Header = () => {
...
-            </ul>
-          </nav>
-        </div>
-      )}
+          </div>
+          <ul className="flex w-full flex-col justify-center gap-12 py-16">
+            {menuItems.slice(0, -1).map(
+              (
+                item,
+                index // Exclude 'PublishYourRole' for mobile
```
Comment (JoaoVictor6) [inline apps/web/app/components/Header.tsx:117]: Why exclude 'PublishYourRole' on the mobile version?

Comment (omateusamaral) [reply apps/web/app/components/Header.tsx:117]: I do not know, this already was here before the pull request. so I decided to continue with this logic.

**Comment by ocodista** [comment]
@omateusamaral That's really nice, thank you for your contribution!!!

**Review by ocodista — APPROVED** [review]

**Comment by ocodista** [comment]
@JoaoVictor6 shouldn't this PR trigger a build?

**Comment by JoaoVictor6** [comment]
> @JoaoVictor6 shouldn't this PR trigger a build?

I haven't merged the workflow file yet

**Review by ocodista — APPROVED** [review]

**Pull request merged**